codegen.ts

```diff
@@ -49,7 +49,7 @@ const config: CodegenConfig = {
     },
     // "lib/graphql/generated/dpl-cms/graphql.schema.json": {
     //   // TODO: Make this configurable
-    //   schema: "http://dapple-cms.docker/graphql",
+    //   schema: "http://dapple-cms.local/graphql",
```

Comment:
Please revert this until we decide to go with `.local`

```suggestion
    //   schema: "http://dapple-cms.docker/graphql",
```